tests/test_dump.py

```diff
@@ -61,3 +61,48 @@ def test_dump_file(tmp_path):
 
 def test_varied_list():
     assert rtoml.dumps({'test': [1, '2']}) == 'test = [1, "2"]\n'
+
+
+@pytest.mark.parametrize(
+    'input_obj,output_toml',
+    [
+        ({'key': None}, ''),
```

Comment:
what about `None` as the key: `{None: 1}`?

Reply:
Good catch.
I did fix it now it doesn't serialize anything.